graphbolt/src/index_select.cc

```diff
@@ -107,10 +107,6 @@ std::tuple<torch::Tensor, torch::Tensor> IndexSelectCSC(
         c10::DeviceType::CUDA, "IndexSelectCSCImpl",
         { return IndexSelectCSCImpl(indptr, indices, nodes, output_size); });
   }
-  // @todo: The CPU supports only integer dtypes for indices tensor.
```

Comment:
Do we need to worry about non-integer indices?

Reply:
We can move the InSubgraph implementation into IndexSelectCSCBatched. Then, we won't have this problem anymore. Will do it in next PR.